plugins/ftdidmx/FtdiWidget.h

```diff
@@ -54,23 +53,33 @@ class FtdiWidgetInfo {
  public:
     FtdiWidgetInfo(const std::string &name,
                    const std::string &serial,
-                   unsigned int id)
+                   unsigned int id,
+                   const uint16_t vid = 0x0403,
```

Comment:
Given these VID and PID are used in a few places, can we make them a constant please.